### DISCUSSION TITLE
3.3.1节“通过变量修改历史记录”操作上的疑问

### DISCUSSION THREAD

**Discussion by bczhc** [Physics365/991CN-X-CW-Decompilation#36] [Q&A] [answered]
> 有一种广为人知的刷字符方法:刷出字符 @...最后`CALC` `AC`,最上面一条历史记录就是想要卡的字符。

可依据我的测试，要加个等于：`CALC` `=` `AC` `↑` `↑`才能成功，而且这样第一条（上翻一次）历史记录算式为`@=1.00002424..23`，答案是`1.00002424..`；第二条（上翻两次）历史为算式`$$$$`，答案是0，并没出现教程中写的`6.894757`。是哪里做错了吗？

**Comment by Physics365** [comment]
没有做错，教程写的不清楚，会改的

**Comment by EwTE-H** [comment, marked as answer]
> 可依据我的测试，要加个等于：`CALC` `=` `AC` `↑` `↑`才能成功

这里实在抱歉，我写的时候疏忽了。
> 第二条（上翻两次）历史答案是0，并没出现教程中写的`6.894757`。

`06 89 47 57 00 00 00 01 00 00`，根据最后一位判断，为“整体正指数负”的情况。指数为00-100=-100，超出了计算器能显示的范围，所以显示成0。这一点也是我疏忽了。

**Reply by bczhc** [reply]
这样啊，感谢解答！
